mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,34 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+import os.path
+
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
```

Comment:
Could you update this message so that it refers to `authorities.json` instead `frontend`?